packages/carbon-react/src/components/Tag/Tag.stories.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { Tag } from 'carbon-components-react';
+
+export default { title: 'Components/Tag' };
```

Comment:
I think we can add in a `component: Tag` field to this for the prop table to work